src/unit_tests/wrappers/wazuh/shared/file_op_wrappers.c

```diff
@@ -245,3 +245,12 @@ char *__wrap_GetRandomNoise() {
 const char *__wrap_getuname() {
     return mock_ptr_type(char*);
 }
+
+char * __wrap_w_get_file_content(__attribute__ ((__unused__)) const char * path,
+                                 __attribute__ ((__unused__)) int max_size) {
+    return mock_type(char *);
```

Comment:
Why the parameters are not checked?

Reply:
I don't know, that wrapper was already done, I only brought it because I didn't take the path where it was defined